core/templates/dev/head/pages/admin/roles_tab/roles_tab_directive.html

```diff
@@ -0,0 +1,98 @@
+<style>
+    .form-horizontal .control-label {
+        text-align: left;
+    }
+</style>
+
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-8">
+      <md-card class="oppia-page-card oppia-long-text">
+        <form class="form-horizontal" name="viewRoleForm" ng-submit="submitRoleViewForm(viewFormValues)">
+          <legend class="text-center">View User Roles</legend>
+          <div class="form-group">
+            <label class="col-md-4 col-lg-4 col-sm-4 control-label pull-left" for="viewFormValues.method">
+              Select Method
+            </label>
+            <div class="col-md-8">
+              <select ng-model="viewFormValues.method" class="form-control">
+                  <option value="role">By Role</option>
+                  <option value="username">By Username</option>
+              </select>
+            </div>
+          </div>
+
+          <div class="form-group" ng-if="viewFormValues.method==='role'">
+            <label class="col-md-4 col-lg-4 col-sm-4 control-label pull-left">Select Role</label>
+            <div class="col-md-8 col-lg-8 col-sm-8">
+              <select ng-options="roleName as roleString for (roleName, roleString) in VIEWABLE_ROLES" ng-model="viewFormValues.role" class="form-control">
+              </select>
+            </div>
+          </div>
+
+          <div class="form-group" ng-if="viewFormValues.method==='username'">
+            <label class="col-md-4 col-lg-4 col-sm-4 control-label">Enter Username</label>
+            <div class="col-md-8 col-lg-8 col-sm-8">
+              <input type="text" name="username" placeholder="Enter username" ng-model="viewFormValues.username" class="form-control" />
+            </div>
+          </div>
+
+          <button type="submit" class="btn btn-success"
+                  ng-disabled="(viewFormValues.method==='role' && !viewFormValues.role) || (viewFormValues.method==='username' && !viewFormValues.username)"
+                  value="view role">
+                  View Role
+          </button>
+        </form>
+
+        <div ng-show="showResultRoles">
+          <br/>
```

Comment:
Try not to use br; that's a bit of an antipattern. Instead, apply margin at the top of the table.